java/hillshade-renderer/README.md

```diff
@@ -1,38 +1,32 @@
-# Hillshade Renderer
-Use a hillshade renderer on a raster.
+# Hillshade renderer
 
-![Hillshade Renderer App](hillshade-renderer.png)
+Apply a hillshade renderer to a raster.
+
+![Image of hillshade renderer](hillshade-renderer.png)
+
+## Use case
+
+An environmental agency may track coastal erosion by comparing images of an area taken over a a longer period of time with hillshade renderers applied.
 
 ## How to use the sample
-Tap on the wrench button in the Action Bar to change the settings for the `HillshadeRenderer`. The sample allows you to change the `Altitude`, `Azimuth` and `Slope type`. You can tap on the `Render` button to update the raster.
+
+Choose and adjust the settings to update the hillshade renderer on the raster layer. The sample allows you to change the Altitude, Azimuth, and Slope Type.
 
 ## How it works
-The sample uses the `HillshadeRenderer` class to render new hillshades. The parameters provided by the user are passed to `HillshadeRender` at instantiation: `new HillshadeRenderer(mAltitude, mAzimuth, mZFactor, mSlopeType, mPixelSizeFactor, mPixelSizePower, mOutputBitDepth);` which returns a `RasterRenderer`. The `RasterRenderer` is then added to the `RasterLayer`.
+
+1. Create a `Raster` from a grayscale raster file.
+2. Create a `RasterLayer` from the raster.
+3. Create a `Basemap` from the raster layer and set it to the map.
+4. Create a `HillshadeRenderer`, specifying the slope type and other properties, `new HillshadeRenderer(Altitude, Azimuth, ZFactor, SlopeType, PixelSizeFactor, PixelSizePower, OutputBitDepth)`.
```

Comment:
We don't need to include the whole class constructor here (since the customer will use the code to understand how it's put together: this readme acts as a quick overview guide). Just the following will suffice:
```suggestion
4. Create a `HillshadeRenderer`, specifying the slope type and other properties.
```